src/decoder_allocators.hpp

```diff
@@ -30,7 +30,7 @@
 #ifndef ZEROMQ_DECODER_ALLOCATORS_HPP
 #define ZEROMQ_DECODER_ALLOCATORS_HPP
```

Comment:
Well, the usual practice seems to be to use something like 
__ZMQ_STREAM_ENGINE_HPP. If I'm not mistaken, using double underscores is not 
allowed in C++ because it is reserved for internal names.

Anyway, this seems to turn into a discussion of style, and the style guide 
doesn't specify most of the points from the other e-mails. I wouldn't create 
rules for some of the points, e.g. I would not define a rule where to put 
spaces in function calls. I would rather say "use spaces to make it readable". 
If I can't read it, I reformat it when reading the code.

Cheers,
  Jens

On Sunday 05 July 2015 14:41:15 Thomas K�ppe wrote:
> > @@ -30,7 +30,7 @@
> > 
> >  #ifndef ZEROMQ_DECODER_ALLOCATORS_HPP
> >  #define ZEROMQ_DECODER_ALLOCATORS_HPP
> 
> The include guard is off. Pieter: Might be worth codifying the existing
> practice in the style guide.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/zeromq/libzmq/pull/1467/files#r33899219